docs/update-from-client-side-helpers-to-server-side.md

```diff
@@ -0,0 +1,107 @@
+    <div id="paypal-button-container"></div>
+    <script>
+      window.paypal
+        .Buttons({
+          // Set up the order
+          createOrder: function (data, actions) {
+-           return actions.order.create({
+-              purchase_units: [
+-                {
+-                   amount: {
+-                      value: "88.44",
+-                    },
+-                 }],
+-                });
++             return (
++               fetch("/your-server/api/create-paypal-order", {
++                 method: "POST",
++                 // use the "body" param to optionally pass additional order information
++                 // like product skus and quantities
++                 body: JSON.stringify({
++                   cart: [
++                     {
++                        sku: "YOUR_PRODUCT_STOCK_KEEPING_UNIT",
++                        quantity: "YOUR_PRODUCT_QUANTITY",
++                      }],
++                   }),
++                 })
++                 .then((response) => response.json())
++                 // return the PayPal Order ID
++                 .then((order) => order.id)
++               );
+            },
+            // Finalize the transaction
+            onApprove: function (data, actions) {
+-             return actions.order
+-               .capture()
+-               .then(function (orderData) {
+-                  // Successful capture!
+-                })
+-                .catch(function (err) {
+-                   // Failed capture
+-                });
++                return fetch("/my-server/patch-paypal-order", {
++                  method: "POST",
++                })
++                .then((response) => response.json())
++                .then(function (orderData) {
++                   // Successful capture!
++                })
++                .catch(function (err) {
++                   // Failed capture
++                });
+            },
+            onError: function (err) {
+              // For example, redirect to a specific error page
+              window.location.href = "/your-error-page-here";
+            }
+        })
+        .render("#paypal-button-container");
+    </script>
+  </body>
+</html>
+```
+
+- What is server-side code?
+
+  Server-side code runs securely on a web server and is typically used to communicate with APIs and Databases. Here's a list of common server-side languages used to make websites: Node.js, PHP, ASP.NET, Ruby, Java.
+
+- What should I do if I do not have the ability to run server-side code?
+
+  We recommend using one of PayPal's partners to host your website like Wix, GoDaddy, Shopify, and Big Commerce.
+
+## Best Practices for JS SDK Server-Side Integrations:
+
+1. CLIENT_SECRET should never be checked into git. We recommend passing this sensitive value to the web server at runtime as an environment variable. It's common to use a .env file that is ignored by git to load sensitive values like this secret.
+Examples:
```

Comment:
Perhaps we link to each languages recommended env file library instead of showing the server start up commends with env variables.

- Node.js: https://github.com/motdotla/dotenv
- PHP: https://github.com/vlucas/phpdotenv
- Python: https://github.com/theskumar/python-dotenv